src/pages/SignUp/SignUp.js

```diff
@@ -1,71 +1,108 @@
-import React from 'react';
+import React, { useState } from 'react';
+import SignUpContainer from './SignUpContainer/SignUpContainer';
 import './SignUp.scss';
-import { Link } from 'react-router-dom';
+
+const INPUT_DATA_INDIVIDUAL = [
+  {
+    id: 1,
+    name: 'name',
+    className: 'inputValue',
+    type: 'text',
+    placeholder: '이름',
+  },
+  {
+    id: 2,
+    name: 'email',
+    className: 'inputValue',
+    type: 'text',
+    placeholder: '이메일 주소',
+  },
+  {
+    id: 3,
+    name: 'pw',
+    className: 'inputValue',
+    type: 'password',
+    placeholder: '비밀번호',
+  },
+];
+
+const INPUT_DATA_BUSINESS = [
+  {
+    id: 1,
+    name: 'name',
+    className: 'inputValue',
+    type: 'text',
+    placeholder: '사업자명',
+  },
+  {
+    id: 2,
+    name: 'num',
+    className: 'inputValue',
+    type: 'text',
+    placeholder: '사업자번호',
+  },
+  {
+    id: 3,
+    name: 'email',
+    className: 'inputValue',
+    type: 'text',
+    placeholder: '이메일 주소',
+  },
+  {
+    id: 4,
+    name: 'pw',
+    className: 'inputValue',
+    type: 'password',
+    placeholder: '비밀번호',
+  },
+];
+
+const BLACK_BTN = {
+  className: 'blackBtn',
+  btnText: '회원가입',
+};
 
 const SignUp = () => {
+  const [signUp, setSignUp] = useState(null);
+  const [signUpBtn, setSignUpBtn] = useState(false);
+
+  const handleButtonClick = buttonName => {
+    setSignUp(buttonName);
+    setSignUpBtn(true);
+  };
+
   return (
     <div className="signUp">
-      <div className="accountSelect">
+      {}
+      <div
+        className="accountSelect"
+        style={{ display: signUpBtn ? 'none' : 'block' }}
+      >
```

Comment:
```suggestion
{
signUpBtn &&       
<div
        className="accountSelect"
        style={{ display: signUpBtn ? 'none' : 'block' }}
      >
      //코드생략
</div>
}
```
위처럼 signUpBtn의 값에 따라 조건부 랜더링을 해준다면 inline-styling을 해주지 않아도 되겠네요!  
앞으로 개발하실때도 inline-styling은 지양해주세요!
그 이유에는 다양한 이유가 있지만 
[참고자료](https://spare8433.tistory.com/21) 한번 읽어보시면 좋겠네요!